### PR TITLE
Add weather app to demos

### PR DESCRIPTION
**Pull request by AJBrownson**
## Description

- Adding a new demo application in the `/demos` folder
- Addressing hacktoberfest issue: 🤩 Create Weather App (Hacktoberfest Demo) [#2566](#2566)
- Elemental is a weather app that gets the current user's location and shows the weather information for that location. It also 
   allows users to search for weather information by city name. It uses the OpenWeather API to retrieve weather data and the
    Tolgee API to provide translations.

### Key Features:
- Text Translation
- Weather Information
- Search by City Name
- Responsive Design

### Screenshots
Weather App Screenshot

### Demo
View the demo live [here](https://elemental-ten.vercel.app)

### Article
Read the DEV article [here](https://dev.to/anni/how-to-build-a-weather-forecast-app-with-react-typescript-tolgee-and-openweather-api-2197)

## Checklist
- [✓] I have read the Contribution Guide
- [✓] If the PR changes or adds functionality, I have updated the relevant documentation
- [✓] Translate all labels in the App with Tolgee
- [✓] Store the project into [/demos](https://github.com/tolgee/tolgee-platform/tree/main/demos) folder
- [✓] Add a screenshot or video demo to the PR
- [✓] Write a short article describing your demo app on DEV.to or your blog
- [✓] Please show your support and give a ⭐ to the Tolgee repository

### CODE REVIEW

File: demos/flagmoji/README.md [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+```
+
+4. Open [http://localhost:3000](http://localhost:3000) in your browser to view the app
+
+5. To host on Netlify or Vercel, run the following command:
+
+```bash
+npx tolgee login <your-tolgee-api-key>
+
+npx tolgee --project-id <your-project-id-number> pull --path public/i18n
```
[the remark this answers is not in the document]
Comment (AJBrownson) [reply demos/flagmoji/README.md:77]: Okay
Should I remove the line from the README file?

File: demos/flagmoji/README.md [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+```
+
+Add the following code to your main.tsx file:
+
+```typescript
```
[the remark this answers is not in the document]
Comment (AJBrownson) [reply demos/flagmoji/README.md:82]: I put that in the README to guide future users
Should I remove that line?

**Comment by AJBrownson** [comment]
@JanCizmar 
I have added the vercel demo link and the dev.to article link also

**Comment by AJBrownson** [comment]
@JanCizmar 
I have made the correct commits
Waiting for your review

**Review by JanCizmar — APPROVED** [review]
Looks fine! Thanks a lot! 🚀

**Comment by marketachalupnikova** [comment]
Hi @AJBrownson ! Thanks again for your Hacktoberfest 2024 contribution! 🎉 Please fill out this survey so we can send your reward!

https://app.formbricks.com/s/cm375z4j6000amd4zgh5b3ycg

**Comment by JanCizmar** [comment]
Thanks for contributing and sorry for the delay. Here is the project evaluation! 🚀

Quality: 	13	(max 20)
User Experience (UX): 	10	(max 10)
Creativity: 	9	(max 10)
Usefulness: 	9	(max 10)
Total: 	41	(max 50)
		
Your prize is	Tolgee Sticker set + Tolgee Socks

**Pull request merged**